int16 target_id 
string target_type
int16 num_attempts
---
bool isConfirmed